projects/cashmere/src/lib/tabs/tab-set/tab-set.component.ts

```diff
@@ -336,28 +348,27 @@ export class TabSetComponent implements AfterContentInit {
         this._tabArrowsEnabled[1] = this._tabBar.nativeElement.offsetWidth === this._tabBar.nativeElement.scrollWidth - this._tabBar.nativeElement.scrollLeft ? false : true;
```

Comment:
This is the line that was not working for me to determine if the right arrow should be enabled or not.
The problem was that, although the scrollLeft value was supposed to be an integer, it wasn't so the `offsetWidth `was never exactly equal to the `this._tabBar.nativeElement.scrollWidth - this._tabBar.nativeElement.scrollLeft`.